sklearn/preprocessing/_data.py

```diff
@@ -683,6 +681,11 @@ class StandardScaler(OneToOneFeatureMixin, TransformerMixin, BaseEstimator):
     than others, it might dominate the objective function and make the
     estimator unable to learn from other features correctly as expected.
 
+    StandardScaler is sensitive to outliers, and the features may scale
+    differently from each other in the presence of outliers. For an example
```

Comment:
I'm not sure if I understand the sentence here, as in yes, it is sensitive to outliers, but I'm not sure about the second part about features being scaled differently from one another.

Reply:
This was my attempt to put into words my main learning from reading the example file:

https://github.com/scikit-learn/scikit-learn/blob/ce41907ce0f0c29c1f224df2dad1b7305d35b8e3/examples/preprocessing/plot_all_scaling.py#L274C1-L279C73

Should we rather use words closer to what was used there? (" Note in particular that because the outliers on each feature have different magnitudes, the spread of the transformed data on each feature is very different")